src/items/functions.md

```diff
@@ -174,12 +174,15 @@ extern "C" fn new_i32() -> i32 { 0 }
 let fptr: extern "C" fn() -> i32 = new_i32;
 ```
 
-Functions with an ABI that differs from `"Rust"` do not support unwinding in the
-exact same way that Rust does. Therefore, unwinding past the end of functions
-with such ABIs causes the process to abort.
+Functions with an ABI that differs from `"Rust"` do not support unwinding in
+the exact same way that Rust does. Therefore, unwinding past the end of
+functions with such ABIs results in undefined behavior, and will differ between
+platforms and compiler editions. If you cannot guarantee this will not happen,
```

Comment:
I'm not sure what "compiler editions" means here.  "Edition" has a specific meaning in Rust, and I don't think that is what is being referred to here.  I think it would be fine to end the sentence at "undefined behavior", with the link leading the reader to understanding what that means.